### PR TITLE
UAF-2097 - Bug - NullPointer exception in KualiHttpSessionListener- Round 2

### PR DESCRIPTION
**Pull request by sskinner**
Adding NPE checks and log messages to ensure logs don't get filled with excessive STE's.

### CODE REVIEW

File: impl/src/main/java/org/kuali/rice/krad/web/listener/KualiHttpSessionListener.java [hunk 1]
```diff
@@ -53,7 +56,11 @@ public void sessionCreated(HttpSessionEvent se) {
     }
 
     /**
-     * HttpSession hook for additional cleanup when sessions are destroyed
+     * HttpSession hook for additional cleanup when sessions are destroyed.
+     *
+     * UAR-2097: Ensure already destroyed sessions can'r casue STE's to fill up logs during
```
Comment (ghost) [inline impl/src/main/java/org/kuali/rice/krad/web/listener/KualiHttpSessionListener.java]: Shouldn't this be UAF-2097 in this comment?

**Comment by ghost** [comment]
Initial review complete. Please see my comment.

File: impl/src/main/java/org/kuali/rice/krad/web/listener/KualiHttpSessionListener.java (hunk 1 above)
Comment (sskinner) [inline impl/src/main/java/org/kuali/rice/krad/web/listener/KualiHttpSessionListener.java]: It absolutely should, KC _still_ stuck on my brain.

**Comment by ghost** [comment]
Looks good!

**Pull request merged**